tools/deb/istio-auth-node-agent.service

```diff
@@ -1,9 +1,9 @@
 [Unit]
-Description=istio-node-agent: The Istio node agent
+Description=istio-auth-node-agent: The Istio auth node agent
 Documentation=http://istio.io/
 
 [Service]
-ExecStart=/usr/local/bin/istio-node-agent --env=0 --alsologtostder
+ExecStart=/usr/local/bin/istio-auth-node-agent
```

Comment:
why are the CMD options being removed? Are they specified somewhere else?

Reply:
'env' is unnecessary since we are using the default value, --alsologtostderr is mostly for debugging purpose.